service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -138,8 +141,11 @@ public void handleSubmittedEvent(@RequestHeader(value = "authorization") String
     private Document getDocument(String authorization,
```

Comment:
Adam has made something that auto injects the authorisation string, thoughts on changing it here to try and use it?

Reply:
Happy to do so in separate PR, did not edit this method.

Reply:
Fair enough, maybe a PR to go through all occurrences and replace it